src/libraries/Common/src/Interop/Windows/Kernel32/Interop.RaiseFailFastException.cs

```diff
@@ -25,23 +25,35 @@ internal unsafe struct EXCEPTION_RECORD
 
     internal partial class Kernel32
     {
-        internal const uint EXCEPTION_NONCONTINUABLE = 0x1;
+        private const uint EXCEPTION_NONCONTINUABLE = 0x1;
 
-        internal const uint FAIL_FAST_GENERATE_EXCEPTION_ADDRESS = 0x1;
+        private const uint FAIL_FAST_GENERATE_EXCEPTION_ADDRESS = 0x1;
+
+        private const uint STATUS_STACK_BUFFER_OVERRUN = 0xC0000409;
+
+        private const uint FAST_FAIL_EXCEPTION_DOTNET_AOT = 72;
 
         //
-        // Wrapper for calling RaiseFailFastException
+        // NativeAOT wrapper for calling RaiseFailFastException
         //
+
         [DoesNotReturn]
-        internal static unsafe void RaiseFailFastException(uint faultCode, IntPtr pExAddress, IntPtr pExContext)
+        internal static unsafe void RaiseFailFastException(uint errorCode, IntPtr pExAddress, IntPtr pExContext, void* pTriageBlock, int cbTriageBlock)
         {
             EXCEPTION_RECORD exceptionRecord;
-            exceptionRecord.ExceptionCode = faultCode;
+            exceptionRecord.ExceptionCode = STATUS_STACK_BUFFER_OVERRUN;
```

Comment:
`STATUS_STACK_BUFFER_OVERRUN` can use comment since it is completely non-obvious why we would report all failfasts as stack buffer overruns.